### DISCUSSION TITLE
Uno Template version 4.3.8.10 Net6 (Few noob basic questions for every body)

### DISCUSSION THREAD

**Discussion by Daantje007** [unoplatform/uno#9169] [Q&A] [answered]
Hello Uno-Team,

so i have my first uno-Net6 working. nice! but what now?
![HKAPPSPEC](https://user-images.githubusercontent.com/95089394/176403815-6ef91f8f-5255-468e-b1ec-48f80aae707c.png)

My first question is: **what do i actualy realy got?**

Windows SDK App (Basicly for shell ) including WinUI3( Decoupled xaml from Windows and the Application) Net6 ( Framework).. this seems like somekind of addons/Innovations, **But what exactly is my App? (and version)**  it quite hard to find a way in the documentions and i am not 100% sure.

The second and third question: if i understand correctly. It is no longer based on Xamarin.  But Uno based on/Including Maui. 

**Can i use the Toolkits.Maui? in uno** and if yes where is my MauwiProject.cs to initialize them??
**Does this also means i cant use the known ComunityToolkids?** like for instance: communitytoolkit.winui.ui.Controls.Primitives (the old goodies) ?

The fourth question: **is it still not needed/posible to update the (Microsoft.Extentions.Logging(.Console))?** because they are depricated. 
and i know from the (_xamarin based one and uwp_) it should stay version 5.0! does this count for the Net6-Template also?

I am a Junior-developer, so i can unterstand for some of you it wil look like stupid questions. and they are.
so please write the awnsers like: Yes or No + Awnser hihi, **Thank you all and goodluck with your projects.**

Also what to do with dis popup and do you like it XD just kidding? 
![Ooops](https://user-images.githubusercontent.com/95089394/176412150-41efbb3f-3497-40c7-a201-602ee5e0f3bd.png)

**Comment by Daantje007** [comment]
Dont say: Yes + never show this again!! XD

**Comment by jeromelaban** [comment, marked as answer]
> My first question is: **what do i actualy realy got?**

You'll have to be more precise.

> Windows SDK App (Basicly for shell ) including WinUI3( Decoupled xaml from Windows and the Application) Net6 ( Framework).. this seems like somekind of addons/Innovations, **But what exactly is my App? (and version)** it quite hard to find a way in the documentions and i am not 100% sure.

What do you mean by "what is my app"? Which version are you talking about ?

> The second and third question: if i understand correctly. It is no longer based on Xamarin. But Uno based on/Including Maui.

This is correct, the default uno template uses .NET 6 Mobile. You can still use the UWP/Xamarin template if you still need to.

> **Can i use the Toolkits.Maui? in uno** and if yes where is my MauwiProject.cs to initialize them?? 

Uno Platform does not use MAUI. Uno uses .NET 6 Mobile, which is the layer below MAUI, which means that most MAUI-specific packages cannot be used.

> **Does this also means i cant use the known ComunityToolkids?** like for instance: communitytoolkit.winui.ui.Controls.Primitives (the old goodies) ?

Yes, you can [use the community toolkit](https://platform.uno/docs/articles/uno-community-toolkit.html?tabs=tabid-winui).

> The fourth question: **is it still not needed/posible to update the (Microsoft.Extentions.Logging(.Console))?** because they are depricated. and i know from the (_xamarin based one and uwp_) it should stay version 5.0! does this count for the Net6-Template also?

Microsoft.Extentions.Logging.Console is not deprecated. If you want to use a later version, you can do so.

> Also what to do with dis popup and do you like it XD just kidding? ![Ooops](https://user-images.githubusercontent.com/95089394/176412150-41efbb3f-3497-40c7-a201-602ee5e0f3bd.png)

This is likely an Android SDK installation error, specific to Visual Studio. You can take a look a Microsoft's documentation for help on this topic.

**Reply by Daantje007** [reply]
Thank you for the great awnsers! 

_What do you mean by "what is my app"? Which version are you talking about ?_

My understanding about Uno woes, the main project from which it all worked in VS woes UWP (Using UWP skill to create Uno apps)
for intelesens you needed to select the UWP head. (LeftUpperCorner of the VS.Xaml page)
and you could mostly folow UWP documentation. UWP uses .Net4.5

I asume this is (.windows) now. so the some what leading project is **not win32/Wpf** also **not winUI2/UWP** but if i am right 
**windows sdk app **v1.00(?)** + winUI3/Windows** 

So when i am looking for howto documentation.. i am looking for **Windows SDK App **v1.00(?)**  or/and  winUI3 (xaml)** 
i think i am confused with WINUI as WPF/UWP  and winUI2, winUI3 libaries.

**Reply by Daantje007** [reply]
Is it posible that the dockpanel is missing even with **communitytoolkit.winui.ui.Controls.Primitives**  installed properly**??** i have got all working on everyting. only the promised dockpanel is not in the list!

**Reply by jeromelaban** [reply]
On which platform do you not see it?

**Comment by Daantje007** [comment]
![Dock](https://user-images.githubusercontent.com/95089394/176632804-87b5ccd1-ff99-4a67-b06f-7db7586daaf6.png)
![Dockpanel](https://user-images.githubusercontent.com/95089394/176620067-849a8731-1bf5-42ba-862a-d668d6d0f9c5.png)

**Reply by Daantje007** [reply]
i assume  no backward compatibility? for just the dockpanel.

**Reply by jeromelaban** [reply]
`DockPanel` is [available here](https://github.com/CommunityToolkit/WindowsCommunityToolkit/blob/winui/CommunityToolkit.WinUI.UI.Controls.Primitives/DockPanel/DockPanel.cs). If you having Community Toolkit specific questions (unless it's about the Uno version), it would be best to [ask the questions here](https://github.com/CommunityToolkit/WindowsCommunityToolkit/discussions).

**Reply by Daantje007** [reply]
DockPanel works now, thks

**Comment by Daantje007** [comment]


**Reply by francoistanguay** [reply]
https://platform.uno/blog/livecharts-announces-support-for-uno-platform/

**Reply by francoistanguay** [reply]
Please let's keep separate topics in separate discussions.
